openedx/core/djangoapps/schedules/templates/schedules/edx_ace/recurringnudge_day10/email/body.txt

```diff
@@ -1,7 +1,14 @@
 {% load i18n %}
+{% if courses|length > 1 %}
 {% blocktrans trimmed %}
-    Keep up the momentum! Many edX learners in {{course_name}} are completing more problems every week, and
+    Many edX learners are completing more problems every week, and
     participating in the discussion forums. What do you want to do to keep learning?
 {% endblocktrans %}
-
-{% trans "Keep learning" %} <{{course_url}}>
\ No newline at end of file
+{% trans "Keep learning" %} <{{dashboard_url}}>
+{% else %}
+{% blocktrans trimmed %}
```

Comment:
nit: indent the contents of your conditional blocks

Reply:
Make sure the manual test plan calls for reviewing the text-based email. Unlike HTML, all whitespace for text emails ends up getting rendered.